### DISCUSSION TITLE
2.10.230331_LTS

### DISCUSSION THREAD

**Discussion by HanFengA7** [LychApe/DreamCat#114] [Announcements]
> ### DreamCat 2.10.230331_LTS  [`下载`](https://github.com/LychApe/DreamCat/archive/refs/tags/2.10.230331_LTS.zip)  
> 1.优化文章页排版  
> 2.优化代码结构  
> 3.去除冗余代码 

## What's Changed
* <fix>: bug by @HanFengA7 in https://github.com/LychApe/DreamCat/pull/108
* 2.x lts by @HanFengA7 in https://github.com/LychApe/DreamCat/pull/113

**Full Changelog**: https://github.com/LychApe/DreamCat/compare/2.9.230208_LTS...2.10.230331_LTS

<hr /><em>This discussion was created from the release <a href='https://github.com/LychApe/DreamCat/releases/tag/2.10.230331_LTS'>2.10.230331_LTS</a>.</em>

**Comment by MaydayV** [comment]
3.0真的简洁好看！点赞👍
